rmw_zenoh_cpp/src/rmw_init.cpp

```diff
@@ -176,6 +176,8 @@ rmw_init(const rmw_init_options_t * options, rmw_context_t * context)
     return RMW_RET_ERROR;
   }
 
+  zc_try_init_log_from_env();
```

Comment:
Please add a comment above here that highlights what this does.

Reply:
Included in becea93